examples/read-a-file.mad

```diff
@@ -0,0 +1,37 @@
+#- import fsPromises from 'fs/promises' -#
```

Comment:
After the rebase I think most of the syntax in the file has slightly changed. Function bodies must now have parens around them, and comments are now `//`